recipes/sniffio/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "sniffio" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2e9b81429e3b7c9e119fcee2673ee3be3229982adc68b3f59317863aba05ebb7
+
+build:
+  number: 0
+  skip: True  # [py<35]
```

Comment:
> We can only do Python >= 3.5 on build and run if the package is noarch.

Ahh OK, that's where I took that from then. 👍 

(This could be noarch but sadly the build condition for `contextvars` prevents that)

Reply:
> This could be noarch but sadly the build condition for contextvars prevents that.

You can do a `noarch` and `Python 3.7`, who uses `3.5` and `3.6` :sunglasses:

Reply:
Not sure I follow, can you please elaborate?

Reply:
I was joking, sorry. You could do a `noarch` but constrained it to `Python >=3.7` ignoring `3.6` and `3.5` users.

Reply:
Oh hehehe. I wish! 😁